src/python/engagement_edge/src/engagement_edge.py

```diff
@@ -72,10 +72,12 @@ def _retrieve_jwt_secret_local(self) -> str:
             try:
                 secretsmanager = boto3.client(
                     "secretsmanager",
-                    region_name="us-east-1",
-                    aws_access_key_id="dummy_cred_aws_access_key_id",
-                    aws_secret_access_key="dummy_cred_aws_secret_access_key",
-                    endpoint_url="http://secretsmanager.us-east-1.amazonaws.com:4584",
+                    region_name=os.environ["AWS_REGION"],
+                    aws_access_key_id=os.environ["SECRETSMANAGER_ACCESS_KEY_ID"],
+                    aws_secret_access_key=os.environ[
+                        "SECRETSMANAGER_ACCESS_KEY_SECRET"
+                    ],
+                    endpoint_url=os.environ["SECRETSMANAGER_ENDPOINT"],
```

Comment:
Personally I'd prefer to see this done as a new factory in 
src/python/grapl-common/grapl_common/env_helpers.py
but not a blocker on accepting

Reply:
Yeah, that's a good idea.